device/samples/NetStandardMultiplexSample/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using System.Threading;
+
+namespace multiplexingDotNetCore
+{
+    class Program
+    {
+        // String containing IotHub Hostname:
+        //"hostname.azure-devices.net"
+        private const string IoTHubHostName = "<iothubhostname>.azure-devices.net";
+
+        //Number of devices for multiplexing
+        private static int MUX_DEVICES = 4;
+        private static DeviceClient[] deviceClients = new DeviceClient[MUX_DEVICES];
+
+        //Array of device Ids (should have the same entries as defined on MUX_DEVICES):
+        //{"device1", "device2", "device3", "device4"}
+        private static String[] deviceIds = new String[] { "<device1>", "<device2>", "<device3>", "<device4>" };
+
+        //Array of device PrimaryKeys (following the same order as above):
+        //{"PrimaryKey device1", "PrimaryKey device2", "PrimaryKey device3", "PrimaryKey device4"}
+        private static String[] devicePrimaryKeys = new String[] { "<PrimaryKey device1>",
```

Comment:
This is really bad practice - at absolutely no point should the user hard-code any Azure keys within C# code.